wlutil/init.py

```diff
@@ -0,0 +1,36 @@
+import logging
+from .wlutil import *
+from .br import br
+
+# Initialize wlutil for the first time
+def oneTimeInit():
```

Comment:
I thought about building the base distros here since it can take a long time and we may as well batch up the high-latency initialization stuff if were running it anyway. It's not strictly needed but might improve the user experience (while slowing down build-setup significantly).